src/app/components/MediaLoader/Placeholder/index.styles.tsx

```diff
@@ -5,11 +5,8 @@ import { focusIndicatorThickness } from '../../ThemeProvider/focusIndicator';
 const styles = {
   placeholder: ({ palette }: Theme) =>
     css({
-      position: 'absolute',
-      top: '0',
-      left: '0',
-      width: '100%',
-      height: '100%',
+      position: 'relative',
+      aspectRatio: '16 / 9',
```

Comment:
@amoore108 please could you check if you haven't already that this is still positioned ok using zoom and text resize?

Reply:
Positioning of the placeholder and button seem ok on zoom and text size changes.